halo2-base/src/gates/range.rs

```diff
@@ -175,6 +223,9 @@ pub trait RangeInstructions<F: ScalarField> {
     }
 
     /// Checks that `a` is in `[0, b)`.
```

Comment:
Is there a limit on how big `b` can be? Is it still safe if `b` is greater than or equal to the max value of `F`?